docs/configuration/monitoring.md

```diff
@@ -58,6 +58,11 @@ Crow is a visualization tool that displays information about the projects and jo
 
 The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
 
+The `crow` tool can be accessed through the `/monitoring` path of the url pointing to your Codacy installation, e.g. `http://codacy.company.org/monitoring`.
```

Comment:
Should't we use the global codacy url instead of `crow.config.codacy.url`? Also it would be cool to pass the `crow.config.crow.url` to a global one for the website to have the monitoring link back.

Reply:
yep.. agree.. that would simplify the solution.
You need to change the base chart tough @joaocosta-codacy

Reply:
good call. 
I am changing this in crow so that the crow.config.codacy.url points to .Values.global.codacy.url in the configmap template